polars/polars-lazy/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -99,13 +98,35 @@ pub(super) fn lowercase(s: &Series) -> PolarsResult<Series> {
     Ok(ca.to_lowercase().into_series())
 }
 
-pub(super) fn contains(s: &Series, pat: &str, literal: bool) -> PolarsResult<Series> {
-    let ca = s.utf8()?;
-    if literal {
-        ca.contains_literal(pat).map(|ca| ca.into_series())
-    } else {
-        ca.contains(pat).map(|ca| ca.into_series())
-    }
+pub(super) fn contains(s: &[Series], literal: bool) -> PolarsResult<Series> {
+    let ca = &s[0].utf8()?;
+    let pat = &s[1].utf8()?;
+
+    let mut out: BooleanChunked = match pat.len() {
+        1 => match pat.get(0) {
+            Some(pat) => {
+                if literal {
+                    ca.contains_literal(pat)?
+                } else {
+                    ca.contains(pat)?
+                }
+            }
+            None => BooleanChunked::full(ca.name(), false, ca.len()),
+        },
+        _ => {
+            let f = |s: &str, pat: &str| {
+                if literal {
+                    s.contains(pat)
+                } else {
+                    Regex::new(pat).unwrap().is_match(s)
```

Comment:
This will panic on a invalid input. I think we should have a `strict=True` argument. 

On `strict` we raise a `ComputeError` on invalid input, on non `strict` we return null if the regex is invalid.

This is what we already do in binary decoding.

Reply:
I made progress toward this direction, however I did not make it until the end...
I didn't know how to handle the error